fs/xfstests.py.data/xfstests.yaml

```diff
@@ -1,41 +1,36 @@
-skip_dangerous: True
 scratch_mnt: '/mnt/scratch'
 test_mnt: '/mnt/test'
 disk_mnt: '/mnt/loop-device'
-# edit group for running specific group like 'quick'
-group:
-# edit test_range for running specific tests like '4,15-20'
-test_range:
+
 loop_type: !mux
     type: 'loop'
-    loop_size: '7GiB'
+    loop_size: '5GiB'
     # Option to provide disk for loop device creation,
     # Uses '/' by default for file creation
     disk: "null"
 fs_type: !mux
     fs_ext4_4k:
         fs: 'ext4'
+        args: '-R xunit generic/001'
         mkfs_opt: '-b 4096'
         mount_opt: '-o block_validity'
-        exclude: "048"
-        gen_exclude: "null"
-        share_exclude: "null"
-    fs_ext4_64k:
-        fs: 'ext4'
-        mkfs_opt: '-b 65536'
-        mount_opt: '-o block_validity'
-        exclude: "048"
-        gen_exclude: "null"
-        share_exclude: "null"
     fs_xfs_4k:
         fs: 'xfs'
+        args: '-R xunit generic/001'
         mkfs_opt: '-f -b size=4096'
-        exclude: "null"
-        gen_exclude: "null"
-        share_exclude: "null"
-    fs_xfs_64k:
-        fs: 'xfs'
-        mkfs_opt: '-f -b size=65536'
-        exclude: "null"
-        gen_exclude: "null"
-        share_exclude: "null"
+        mount_opt: ''
+    fs_btrfs_4k:
+        fs: 'btrfs'
+        args: '-R xunit generic/001'
+        mkfs_opt: '-f -s 4096 -n 4096'
+        mount_opt: ''
+    fs_btrfs_4k:
+        fs: 'btrfs'
+        args: '-R xunit generic/001'
+        mkfs_opt: '-f -s 4096 -n 4096'
+        mount_opt: ''
+    fs_btrfs_64k:
+        fs: 'btrfs'
+        args: '-R xunit generic/001'
+        mkfs_opt: '-f -s 65536 -n 65536'
+        mount_opt: ''
```

Comment:
btrfs_4k section is duplicate and btrfs_64k should be removed if this yaml is only for 4k configuration.
filename also needs to be changed accordingly.